perf.groovy

```diff
@@ -0,0 +1,32 @@
+// Import the utility functionality.
+
+import jobs.generation.*;
+
+def project = GithubProject
+def branch = GithubBranchName
+def projectFolder = Utilities.getFolderName(project) + '/' + Utilities.getFolderName(branch)
+
+
+[true, false].each { isPR ->
+    ['Windows_NT'].each { os ->
+        def newJob = job(Utilities.getFullJobName(project, "perf_${os}", isPR)) {
+            // Set the label.
+            steps {
+                    // Batch
+                    batchFile("set __TestIntermediateDir=int&&build.cmd release x64")
```

Comment:
Instead of having many different batch file calls, can we hoist this out into one cmd file under tests/scripts as we have done with other jobs?